.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @gsoft-inc/tech-platform
```

Comment:
Should it be `@gsoft-inc/internal-developer-platform`?

Reply:
ahh men, that copy pasta